<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="216" started-at="2020-08-24T02:27:31Z" finished-at="2020-08-24T02:27:31Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="216" started-at="2020-08-24T02:27:31Z" finished-at="2020-08-24T02:27:31Z">
      <class name="org.automation.weather.utilities.Reports">
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:org.automation.weather.utilities.Reports@47f6473]" name="loginTest" duration-ms="198" started-at="2020-08-24T07:57:31Z" finished-at="2020-08-24T07:57:31Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[testName cannot be null or empty]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: testName cannot be null or empty
	at com.aventstack.extentreports.ExtentTest.<init>(ExtentTest.java:88)
	at com.aventstack.extentreports.ExtentTest.<init>(ExtentTest.java:109)
	at com.aventstack.extentreports.ExtentReports.createTest(ExtentReports.java:231)
	at com.aventstack.extentreports.ExtentReports.createTest(ExtentReports.java:247)
	at org.automation.weather.utilities.Reports.loginTest(Reports.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- org.automation.weather.utilities.Reports -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
